### PR TITLE
Outline for Diarrheal Disease GBD 2017 Model

### PR DESCRIPTION
**Pull request by aflaxman**
Just an outline, comments welcome

### CODE REVIEW

File: docs/source/models/gbd2017/causes/diarrheal_diseases.rst [hunk 1]
```diff
@@ -4,6 +4,70 @@
...
+   [CDC]_, [Wikipedia]_, [GBD-2017-YLD-Capstone-Appendix-1]_
+
+The relevant ICD codes are...
+
+Modeling Diarrheal Diseases in GBD 2017
+---------------------------------------
+
+.. todo::
+
+   Add relevant detail about measles modeling process from
```
Comment (ctroeger) [inline docs/source/models/gbd2017/causes/diarrheal_diseases.rst]: Measles?

File: docs/source/models/gbd2017/causes/diarrheal_diseases.rst [hunk 2]
```diff
@@ -4,6 +4,70 @@
...
+
+.. [Wikipedia] Diarrhea. From Wikipedia, the Free Encyclopedia.
+   Retrieved 14 Nov 2019.
+   https://en.wikipedia.org/wiki/Diarrhea
+
+.. [GBD-2017-YLD-Capstone-Appendix-1]
+   Supplement to: `GBD 2017 Disease and Injury Incidence and Prevalence
+   Collaborators. Global, regional, and national incidence, prevalence, and
+   years lived with disability for 354 diseases and injuries for 195 countries
+   and territories,    Disease Study 2017. Lancet 2018; 392: 178   (pp. 246-7)
```
Comment (NathanielBlairStahn) [inline docs/source/models/gbd2017/causes/diarrheal_diseases.rst]: The appendix pages for diarrhoeal diseases are 88-94, not 246-7 (measles).
```suggestion
   and territories,    Disease Study 2017. Lancet 2018; 392: 178   (pp. 88-94)
```

File: docs/source/models/gbd2017/causes/diarrheal_diseases.rst (hunk 1 above)
Comment (NathanielBlairStahn) [reply docs/source/models/gbd2017/causes/diarrheal_diseases.rst]:
```suggestion
   Add relevant detail about diarrheal diseases modeling process from
```

Comment (aflaxman) [reply docs/source/models/gbd2017/causes/diarrheal_diseases.rst]: woah, cool, i've got to learn how to use suggested changes like this.

**Pull request merged**